src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -52,22 +59,50 @@ public static Dataset datasetRequestToDataset(DatasetRequestModel datasetRequest
                 assetSpecifications.add(assetModelToAssetSpecification(asset, tablesMap, relationshipsMap)));
         }
 
+        CloudPlatform cloudPlatform = Optional.ofNullable(datasetRequest.getCloudPlatform())
+            .orElse(DEFAULT_CLOUD_PLATFORM);
+
+        final List<StorageResource> storageResources;
+        if (cloudPlatform == CloudPlatform.GCP) {
+            storageResources = createGcpStorageResourceValues(datasetRequest.getRegion());
+        } else {
+            throw new UnsupportedOperationException(cloudPlatform + " is not a valid Cloud Platform");
+        }
+
         return new Dataset(new DatasetSummary()
                 .name(datasetRequest.getName())
                 .description(datasetRequest.getDescription())
+                .storage(storageResources)
                 .defaultProfileId(defaultProfileId))
                 .tables(new ArrayList<>(tablesMap.values()))
                 .relationships(new ArrayList<>(relationshipsMap.values()))
                 .assetSpecifications(assetSpecifications);
     }
 
-    public static DatasetSummaryModel datasetSummaryModelFromDatasetSummary(DatasetSummary dataset) {
+    private static List<StorageResource> createGcpStorageResourceValues(String providedRegion) {
+        final GoogleRegion region = Optional.ofNullable(providedRegion)
+            .map(s -> GoogleRegion.fromValue(s.toLowerCase()))
+            .orElse(GoogleRegion.getDefaultGoogleRegion());
+        return Arrays.stream(GoogleCloudResource.values()).map(resource -> {
+            // TODO: Firestore will always be in us-central1 for now, but will likely change in the future.
+            GoogleRegion dbRegion = (resource.equals(GoogleCloudResource.FIRESTORE)) ?
+                GoogleRegion.getDefaultGoogleRegion() :
```

Comment:
Nevermind my previous comment...I guess it makes sense to just have the one default